Can't find 0.10.2.1-cp1 archetype

```
mvn archetype:generate -DarchetypeGroupId=io.confluent.maven -DarchetypeArtifactId=kafka-connect-quickstart -DarchetypeVersion=0.10.2.1-cp1

[INFO] Scanning for projects...
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building Maven Stub Project (No POM) 1
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] >>> maven-archetype-plugin:3.0.1:generate (default-cli) > generate-sources @ standalone-pom >>>
[INFO]
[INFO] <<< maven-archetype-plugin:3.0.1:generate (default-cli) < generate-sources @ standalone-pom <<<
[INFO]
[INFO] --- maven-archetype-plugin:3.0.1:generate (default-cli) @ standalone-pom ---
[INFO] Generating project in Interactive mode
[INFO] Archetype repository not defined. Using the one from [io.confluent.maven:kafka-connect-quickstart:0.10.0.0] found in catalog remote
Downloading: https://repo.maven.apache.org/maven2/io/confluent/maven/kafka-connect-quickstart/0.10.2.1-cp1/kafka-connect-quickstart-0.10.2.1-cp1.pom
[WARNING] The POM for io.confluent.maven:kafka-connect-quickstart:jar:0.10.2.1-cp1 is missing, no dependency information available
Downloading: https://repo.maven.apache.org/maven2/io/confluent/maven/kafka-connect-quickstart/0.10.2.1-cp1/kafka-connect-quickstart-0.10.2.1-cp1.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 9.717 s
[INFO] Finished at: 2017-07-21T10:38:44+10:00
[INFO] Final Memory: 14M/219M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-archetype-plugin:3.0.1:generate (default-cli) on project standalone-pom: The desired archetype does not exist (io.confluent.maven:kafka-connect-quickstart:0.10.2.1-cp1) -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

--- a/README.md
+++ b/README.md
@@ -1,9 +1,11 @@
 This maven quickstart is used to generate a skeleton plugin for Kafka Connect. 
 
 ```
-mvn archetype:generate -DarchetypeGroupId=io.confluent.maven -DarchetypeArtifactId=kafka-connect-quickstart -DarchetypeVersion=0.10.2.1-cp1
+mvn archetype:generate -DarchetypeGroupId=com.github.jcustenborder.kafka.connect -DarchetypeArtifactId=kafka-connect-quickstart -DarchetypeVersion=0.10.2.1-cp2
 ```
 
 ```
-mvn archetype:generate -DarchetypeGroupId=io.confluent.maven -DarchetypeArtifactId=kafka-connect-quickstart -DarchetypeVersion=0.10.2.1-cp1 -Dpackage=io.confluent.examples -DgroupId=io.confluent.examples -DartifactId=testconnect -DpackageName=io.confluent.examples -Dversion=1.0-SNAPSHOT
+mvn archetype:generate -DarchetypeGroupId=com.github.jcustenborder.kafka.connect -DarchetypeArtifactId=kafka-connect-quickstart -DarchetypeVersion=0.10.2.1-cp2 -Dpackage=io.confluent.examples -DgroupId=io.confluent.examples -DartifactId=testconnect -DpackageName=io.confluent.examples -Dversion=1.0-SNAPSHOT
 ```
+
+See https://search.maven.org/#search%7Cga%7C1%7Ca%3A%22kafka-connect-quickstart%22 for a list of available archetypes

